### PR TITLE
Create eoep.json

### PR DESCRIPTION
**Pull request by drawntotheflame**

### CODE REVIEW

**Comment by drawntotheflame** [comment]
@zzorba this is the second PR. Once this is merged, one of us can add Norman's new sigs to the PR with him.

**Review by zzorba — APPROVED** [review]
One correction on Dragon Pole, but looks correct otherwise.

File: pack/eoe/eoep.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+		"slot": "Hand",
+		"text": "[fast] Exhaust Astronomical Atlas: Look at the top card of your deck. If it is not a weakness, attach it facedown to Astronomical Atlas (max 5 cards attached).\n[fast]: Commit a card attached to Astronomical Atlas to an eligible skill test. If that test succeeds, add that card to your hand instead of discarding it. (Limit once per test.)",
+		"traits": "Item. Tome.",
+		"type_code": "asset",
+		"xp": 3
+  },
+  {
+		"code": "08077",
+		"deck_limit": 2,
+		"faction_code": "survivor",
```
Comment (zzorba) [inline pack/eoe/eoep.json:93]: Missing flavor text here.

**Comment by drawntotheflame** [comment]
@Kamalisk this one too, once you've done the other one. Thanks.

**Pull request merged**